src/socket/handler.js

```diff
@@ -139,5 +141,8 @@ export default function setupSocketHandler (app, options, { feathersParams, prov
     socket.on('authenticate', authenticate);
     socket.on(disconnect, logout);
     socket.on('logout', logout);
+
+    entityService.on('updated', updateEntity);
```

Comment:
The only thing I'm wondering about is whether we should do

```js
app.io.on(`${entity} updated`, updateEntity)
app.io.on(`${entity} patched`, updateEntity)
```

so that filters don't get applied. Since this is an internal listener we probably don't want any filters that the developer may have registered to apply here. Otherwise they might be inadvertently preventing an update.

However, there is also a trade-off with that. If we don't use the `app.service()` method then we can't support listening to any remote-decoupled services. In practice that might not be all that realistic and my idea of being able to make auth a standalone service separate from the entity might not actually be possible. Usually a `users` service and `auth` service live on the same app instance.